prep-css-display/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
+    <title>CSS Display Properties</title>
+
+    <div>
+      I am a div and by default I am block, which means I take up all the space
+      I can horizontally.
+    </div>
+
+    <div>
+      I am a div but I have <span>span elements inside me</span>. Span elements
+      are <span>display inline by default</span>, so they only take up as much
+      as space as they need.
+    </div>
+
+    <div>
+      <button>Hello!</button>
+      <button>Buttons are</button>
+      <button>Inline Block</button>
+      <button>By Default</button>
+    </div>
+
+    <ul>
+      <li>Home</li>
+      <li>Products</li>
+      <li>About Us</li>
+      <li>Contact Us</li>
+    </ul>
```

Comment:
In general we want to put visible content in the `<body></body>` of the document and meta information such as title, meta tags, and style elements in the `<head></head>` of the document.

HTML can be very forgiving, but we have to be careful to organize the document in a conventional way for the sake of teammates.